CMakeLists.txt

```diff
@@ -239,12 +239,18 @@ endif()
 
 add_subdirectory(core)
 
-option(Cabana_ENABLE_CAJITA "Build Cajita (grid and particle-grid)" ${Cabana_ENABLE_MPI})
-if(Cabana_ENABLE_CAJITA)
+option(Cabana_ENABLE_GRID "Build grid and particle-grid capabilities" ${Cabana_ENABLE_MPI})
+option(Cabana_ENABLE_CAJITA "Build grid and particle-grid capabilities" OFF)
+if (Cabana_ENABLE_CAJITA)
+  message(DEPRECATION "Option Cabana_ENABLE_CAJITA is deprecated, replaced by Cabana_ENABLE_GRID")
+  set(Cabana_ENABLE_GRID ON)
```

Comment:
The more cmake'ish way would be to use `cmake_dependent_option()`, but as we will ultimately remove this block, I am not it is worth it!

Reply:
Good to know!